Not comptible with react 15.2.1!

Could you please update your awesome plugin?!

thanks!

after doing npm install, i'm getting this error

```
npm WARN peerDependencies The peer dependency react-dom@^15.0.0 || ^0.14.0 included from react-swipeable-views will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency react@^15.3.1 included from react-dom will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--save" "react-swipeable-views" "react-motion"
npm ERR! node v4.4.4
npm ERR! npm  v2.15.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react@15.2.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-native@0.31.0 wants react@~15.2.1
npm ERR! peerinvalid Peer react-redux@4.4.5 wants react@^0.14.0 || ^15.0.0-0
npm ERR! peerinvalid Peer react-dom@15.3.1 wants react@^15.3.1
npm ERR! peerinvalid Peer react-motion@0.4.4 wants react@>=0.13.2 || ^0.14 || ^15.0.0
npm ERR! peerinvalid Peer react-swipeable-views@0.7.1 wants react@^15.0.0 || ^0.14.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dantecervantes/Documents/react-native/MiDiocesis/npm-debug.log
iMac-de-Dante:MiDiocesis dantecervantes$ 
```

--- a/README.md
+++ b/README.md
@@ -13,8 +13,16 @@
 
 ## Installation
 
+### Browser
+
 ```sh
 npm install --save react-swipeable-views react-motion
+```
+
+### Native
+
+```sh
+npm install --save react-swipeable-views
 ```
 
 ## The problem solved

--- a/package.json
+++ b/package.json
@@ -34,9 +34,7 @@
     "warning": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^15.0.0 || ^0.14.0",
-    "react-dom": "^15.0.0 || ^0.14.0",
-    "react-motion": "^0.4.0"
+    "react": "^15.0.0 || ^0.14.0"
   },
   "files": [
     "lib",

